lib/index.js

```diff
@@ -13,7 +12,7 @@ var presentResponses = function(jsonStrings) {
 };
 
 var getOpts = function (searchQuery, options) {
-  var baseUrl = options.baseUrl || SPRINTLY_URL;
+  var baseUrl = options.baseUrl;
```

Comment:
If you change this to:

```js
options = _.defaults(options, {baseUrl: SPRINTLY_URL});
var baseUrl = options.baseUrl;
```

It will have the behavior of being blank if you set it to `''` but not blank if you forget to add it.